backend/app/main.py

```diff
@@ -212,6 +274,8 @@ async def get_document(document_id: str, session: Session = Depends(get_session)
             DistrictrMap.tiles_s3_path.label("tiles_s3_path"),  # pyright: ignore
             DistrictrMap.num_districts.label("num_districts"),  # pyright: ignore
             DistrictrMap.extent.label("extent"),  # pyright: ignore
+            # get total population based on parent layer
+            population_subquery.as_scalar().label("total_population"),
```

Comment:
H: It feels really heavy to calculate the total population here in a subquery. Especially since this is a frequently hit endpoint. I'd suggest adding `total_population` to `DistrictrMap` and getting the column in the join w/ `DistrictrMap.total_population.label("total_population")`

Reply:
confirmed not needed now and removed in favor of changes in #87